template/cordova/lib/log.js

```diff
@@ -0,0 +1,227 @@
+        if (getElementValue(event, './System/Channel') === 'Microsoft-Windows-AppHost/Admin' &&
+            typeof getElementValue(event, './UserData/WWAUnhandledApplicationException') === 'undefined') {
+            return;
+        }
+
+        var result = {
+            channel: getElementValue(event, './System/Channel'),
+            timeCreated: getElementValue(event, './System/TimeCreated', 'SystemTime'),
+            pid: getElementValue(event, './System/Execution', 'ProcessID'),
+            proc: getElementValue(event, './UserData/WWADevToolBarLog/DisplayName'),
+            source: getElementValue(event, './UserData/WWADevToolBarLog/Source'),
+            documentFile: getElementValue(event, './UserData/WWADevToolBarLog/DocumentFile') ||
+                          getElementValue(event, './UserData/WWAUnhandledApplicationException/DocumentFile'),
+            line: getElementValue(event, './UserData/WWADevToolBarLog/Line'),
+            column: getElementValue(event, './UserData/WWADevToolBarLog/Column'),
+            sourceFile: getElementValue(event, './UserData/WWAUnhandledApplicationException/SourceFile'),
+            sourceLine: getElementValue(event, './UserData/WWAUnhandledApplicationException/SourceLine'),
+            sourceColumn: getElementValue(event, './UserData/WWAUnhandledApplicationException/SourceColumn'),
+            message: getElementValue(event, './UserData/WWADevToolBarLog/Message'),
+            displayName: getElementValue(event, './UserData/WWAUnhandledApplicationException/DisplayName'),
+            appName: getElementValue(event, './UserData/WWAUnhandledApplicationException/ApplicationName'),
+            errorType: getElementValue(event, './UserData/WWAUnhandledApplicationException/ErrorType'),
+            errorDescription: getElementValue(event, './UserData/WWAUnhandledApplicationException/ErrorDescription'),
+            stackTrace: getElementValue(event, './UserData/WWAUnhandledApplicationException/StackTrace'),
+        };
+ 
+        results.push(result);
+    });
+    
+    return results;
+}
+
+function formatField(event, fieldName, fieldShownName, offset) {
+    var whitespace = '', // to align all field values
+        multiLineWhitespace = ' '; // to align multiline fields (i.e. Stack Trace) correctly
+    for (var i = 0; i < offset; i++) {
+        if (i >= fieldShownName.length) {
+            whitespace += ' ';
+        }
+        multiLineWhitespace += ' ';
+    }
+
+    if (event.hasOwnProperty(fieldName) && (typeof event[fieldName] !== 'undefined')) {
+        event[fieldName] = event[fieldName].replace(/\n\s*/g, '\n' + multiLineWhitespace);
+        return ('\n' + fieldShownName + ':' + whitespace + event[fieldName]).replace(/\n$/m, '');
+    }
+    return '';
+}
+
+function stringifyEvent(event) {
+    if (typeof event === 'undefined') {
+        return;
+    }
+
+    var result = '',
+        offset = 18;
+
+    result += formatField(event, 'channel', 'Channel', offset);
```

Comment:
I think having a simplified CSV event format is more readable as one needs to scroll through lots of console messages than field:value format you are currently using.